### ISSUE TITLE
Getting Started

### ISSUE THREAD

**Issue by MaxTopping** [stanimirivanovde/flight-search#1]
Hi, I came across this repo and would like to try it out. However, I am not the best programmer and am having trouble running this. When I run `./run.sh -f src/main/resources/test-2020.json -s momondo` i get this out put `./run.sh: line 1: gradle: command not found`.
Do I need to install gradle or build/compile the code before running?

**Comment by stanimirivanovde** [comment]
@MaxTopping, yes you need to install gradle. Give it a try again after you install it. Also make sure your gradle executable is in your path so you can call it from anywhere.
